### ISSUE TITLE
Syntax highlighting inconstant within @if tags after latest Atom update.

### ISSUE THREAD

**Issue by kevinPMK** [demyte/language-cshtml#10]
Hey there,

I recently updated my Atom and noticed that the highlighting was kind out of whack. You can see the issue in the photo. The main problem is inside of razer if statements, especially nested ones.

<img width="330" alt="screen shot 2017-02-10 at 11 10 36 am" src="https://cloud.githubusercontent.com/assets/23705435/22834080/003af2fa-ef82-11e6-81c2-9077dddfdb21.png">

**Comment by noccer** [comment]
Yes I too have noticed this issue since the latest Atom update. Up to now this was a terrific package, and even if it can't be fixed, thanks for building it.
